src/util/ChartUtils.ts

```diff
@@ -674,6 +675,9 @@ export type AxisPropsNeededForTicksGenerator = {
   isCategorical?: boolean;
   categoricalDomain?: ReadonlyArray<AxisTick>;
   type?: 'number' | 'category';
+  /**
+   * The range appears to be only used in Angle Axis - needs further investigation
```

Comment:
I would expect all charts to have both a domain and a range, but maybe thats just my secondary school math classes speaking 😅. I know everything is a "domain" in recharts

Reply:
In all categorical charts, this variable is undefined. Perhaps it's computed somewhere else, or it's included in the scale already? Investigation continues

Reply:
Nah I'm just saying from an external perspective, like d3 lets you define a domain and a range. Idt it does anything in recharts besides what you say